.github/workflows/CompatHelper.yml

```diff
@@ -0,0 +1,45 @@
+name: CompatHelper
+on:
+  schedule:
+    # Check for dep updates; run at midnight on the first day of every month
+    - cron: 0 0 1 * *
+  workflow_dispatch:
+permissions:
+  contents: write
+  pull-requests: write
+jobs:
+  CompatHelper:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if Julia is already available in the PATH
+        id: julia_in_path
+        run: which julia
+        continue-on-error: true
+      - name: Install Julia, but only if it is not already available in the PATH
+        uses: julia-actions/setup-julia@v1
```

Comment:
update this to `setup-julia@v2`? Or let the dependabot flag it for updating?

Reply:
I guess we can just change it here and now.